packages/backend/src/api/controllers/status/view/StatusPage.tsx

```diff
@@ -0,0 +1,78 @@
+import { UnixTime } from '@l2beat/common'
+import React from 'react'
+
+import { Status } from '../Status'
+import { Page } from './Page'
+import { reactToHtml } from './renderHtml'
+
+export interface StatusPageProps {
+  title: string
+  statuses: Status[]
+}
+
+export function renderStatusPage(props: StatusPageProps) {
+  return reactToHtml(<StatusPage {...props} />)
+}
+
+export function StatusPage({ title, statuses }: StatusPageProps) {
+  return (
+    <Page title={title}>
+      <h1>{title}</h1>
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Timestamp</th>
+            <th>Date</th>
+            <th>Latest value</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          {statuses.map((status, i) => (
+            <tr key={i} style={{ color: status.isSynced ? undefined : 'red' }}>
+              <td>{status.name}</td>
+              <td>{status.timestamp ?? '-'}</td>
+              <td>{timestampToIsoString(status.timestamp) ?? '-'}</td>
```

Comment:
If `timestamp` was `UnixTime`:
```suggestion
<td>{status.timestamp?.toDate().toISOString() ?? '-'}</td>
```